packages/astro/src/core/app/common.ts

```diff
@@ -1,6 +1,9 @@
 import type { SSRManifest, SerializedSSRManifest, RouteInfo } from './types';
 import { deserializeRouteData } from '../routing/manifest/serialization.js';
 
+export const pagesVirtualModuleId = '@astrojs-pages-virtual-entry';
+export const resolvedPagesVirtualModuleId = '\0' + pagesVirtualModuleId;
```

Comment:
Didn't you say these variables are the ones you need access to? If so they need to be re-exported in app/index.ts